src/Unsafe/MutableArray.hs

```diff
@@ -5,29 +5,24 @@
 -- |
 -- Module: MutableArray
 -- Description: Unsafe wrappers around `MutableArray#`s
-
-module Data.Linear.Internal.MutableArray
-  ( unsafeNewMutArr
-  , readMutArr
-  , writeMutArr
+module Unsafe.MutableArray
```

Comment:
We may want to mark it as internal nonetheless. But it doesn't need to be done now.